### ISSUE TITLE
The status does not change to "ACTIVATED" after API activation

### ISSUE THREAD

**Issue by otakupahp** [drazenbebic/license-manager-for-woocommerce#59]
I call the API to activate the license and although the activations counter increments, the status is still "Send"

![imagen](https://user-images.githubusercontent.com/26227907/73596449-30d00380-44f0-11ea-9a3c-1697c6143197.png)

I have to change manually to "Activated"

**Comment by drazenbebic** [comment]
@otakupahp 

There is no "ACTIVATED" status, there only is the "Active" status.

"Active" means that the license key is available for sale. The status is not meant to change after the final delivery.

**Comment by otakupahp** [comment]
Sorry, bad English hehe

However, that causes confusion. One thing is "Available for sale" and another different one is "Active".

I imagine that active means that the license was activated by the user.

If you won't add that functionality, could you at least name status differently?

Best regards,
Pablo

**Comment by drazenbebic** [comment]
@otakupahp 

Yes, I can do that. I'll add it as a feature request for the next release.

**Comment by drazenbebic** [comment]
@otakupahp 

This turned out to be a bit more work than anticipated. I don't see any urgency with this task at the moment, but I also think that it should be done.

I am removing it from the 2.3.0 milestone, but keeping it as an active issue.

**Comment by AdrianMarikar** [comment]
@drazenbebic 

A question re above. When a license key is set to active, does this mean a license generator will reuse this key?

I currently have edited the plugin to change this to active once the key has been activated. I'm wondering will this tell the generator that an active license key can be used again?

Thanks for the great plugin by the way.

**Comment by drazenbebic** [comment]
Hello @AdrianMarikar 

Generators don't touch existing license keys. "ACTIVE" means that the key is ready for sale.

**Comment by alex-drocks** [comment]
I had a first impression that activating the license should make the status ACTIVES automatically but after thinking more, I actually like the fact that activating does not make ACTIVE status automatically. 
The benefits of not doing it automatically is that you can disabled an abusive license user and not alloe new activations for that key when it is on the DISABLED status. Great thing.

As intended, maybe a documentation precision or renaming is perfectly fine. Thanks again
